src/measure_theory/integral/bochner.lean

```diff
@@ -489,7 +490,8 @@ simple_func.integral_congr (simple_func.integrable f) h
 lemma integral_add (f g : α →₁ₛ[μ] E) : integral (f + g) = integral f + integral g :=
 set_to_L1s_add _ (λ _ _, weighted_smul_null) weighted_smul_union _ _
 
-lemma integral_smul (c : 𝕜) (f : α →₁ₛ[μ] E) :
+lemma integral_smul [normed_ring R] [module R E] [smul_comm_class ℝ R E] [has_bounded_smul R E]
```

Comment:
Unfortunately this isn't the "real" `integral_smul` that I can actually use on real integrals; but it seems like generalizing this lemma is a stepping stone to generalizing the one I actually want!